certif/app/components/session-supervising/candidate-in-list.js

```diff
@@ -193,4 +202,8 @@ export default class CandidateInList extends Component {
     const theoricalEndDateTime = dayjs(this.args.candidate.theoricalEndDateTime).format('HH:mm');
     return theoricalEndDateTime;
   }
+
+  get _hasCertificationOngoingLiveAlert() {
+    return this.args.candidate.liveAlertStatus !== 'ongoing';
```

Comment:
**remarque :** 

Avec ce nom de méthode je m'attend à un === et non pas à un !==

Reply:
du coup le if devient `if (!this._hasCertificationOngoingLiveAlert)`